index.js

```diff
@@ -13,6 +13,7 @@ var defaultOpts = {
   ignored: [],
   ignoreInitial: true,
   queue: true,
+  useFsEvents: false,
```

Comment:
alternative we could sniff version number and only apply this extra option for OSX + >=12.x

Reply:
I like sniffing. Might even want to include the patch version. We have some patterns for that so I'll look after work.